bot/exts/evergreen/trivia_quiz.py

```diff
@@ -421,30 +502,29 @@ def contains_correct_answer(m: discord.Message) -> bool:
 
                 await self.send_answer(
                     ctx.channel,
-                    answers,
+                    quiz_entry.answers,
                     True,
                     question_dict,
-                    self.question_limit - len(done_question) + 1,
+                    self.question_limit - len(done_questions),
                 )
                 await self.send_score(ctx.channel, self.game_player_scores[ctx.channel.id])
 
                 await asyncio.sleep(2)
 
     def make_start_embed(self, category: str) -> discord.Embed:
         """Generate a starting/introduction embed for the quiz."""
+        rules = "\n".join([f"`{index}`: {rule}" for index, rule in enumerate(RULES, start=1)])
```

Comment:
This is minor, but I'm not sure why the rules index is kept in a code block format. It's very odd visually.
![image](https://user-images.githubusercontent.com/20641196/132100818-6a23945d-0634-417c-abb5-5ea25be5e23b.png)